packages/opentelemetry-exporter-jaeger/src/jaeger.ts

```diff
@@ -38,7 +38,17 @@ export class JaegerExporter implements SpanExporter {
       typeof config.flushTimeout === 'number' ? config.flushTimeout : 2000;
 
     config.host = config.host || process.env.JAEGER_AGENT_HOST;
-
+    config.endpoint = config.endpoint || process.env.JAEGER_ENDPOINT;
+    config.username = config.username || process.env.JAEGER_USER;
+    config.password = config.password || process.env.JAEGER_PASSWORD;
+    // https://github.com/jaegertracing/jaeger-client-node#environment-variables
+    // By default, the client sends traces via UDP to the agent at localhost:6832. Use JAEGER_AGENT_HOST and
+    // JAEGER_AGENT_PORT to send UDP traces to a different host:port. If JAEGER_ENDPOINT is set, the client sends traces
+    // to the endpoint via HTTP, making the JAEGER_AGENT_HOST and JAEGER_AGENT_PORT unused. If JAEGER_ENDPOINT is secured,
+    // HTTP basic authentication can be performed by setting the JAEGER_USER and JAEGER_PASSWORD environment variables.
+    this._sender = config.endpoint
+      ? new jaegerTypes.HTTPSender(config)
```

Comment:
If you use an http sender without excluding it from tracing, you can get into a loop where opentelemetry is tracing the send-span requests, thereby creating more spans to send.

Reply:
I got around it by setting:
```typescript
this.provider = new node_1.NodeTracerProvider({
            logger: logger,
            plugins: {
                http: {
                    enabled: this.env.enableHTTPPlugin,
                    path: '@opentelemetry/plugin-http',
                    // @ts-ignore actually 'ignoreOutgoingUrls' is the right parameter, see https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-plugin-http#http-plugin-options
                    ignoreOutgoingUrls: [this.env.endpoint]
                },
                https: {
                    enabled: this.env.enableHTTPSPlugin,
                    path: '@opentelemetry/plugin-https',
                    // @ts-ignore actually 'ignoreOutgoingUrls' is the right parameter, see https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-plugin-https#https-plugin-options
                    ignoreOutgoingUrls: [this.env.endpoint]
                },
                ioredis: { enabled: this.env.enableRedisPlugin, path: '@opentelemetry/plugin-ioredis' },
                mongodb: { enabled: this.env.enableMongoPlugin, path: '@opentelemetry/plugin-mongodb' },
                express: {
                    enabled: this.env.enableExpressPlugin,
                    path: '@opentelemetry/plugin-express'
                },
                dns: { enabled: this.env.enableDNSPlugin, path: '@opentelemetry/plugin-dns' }
            }
        });```

Reply:
Should I add it to the documentation?

Reply:
No, you need to add the `x-opentelemetry-outgoing-request` header to the outgoing request to have it be ignored by tracers. You can see zipkin exporter for an example.

Reply:
@dyladan Done in 256eb0e910a12abb3232043e500def4e38c41b35

Reply:
I also added tests to verify the correct behavior: https://github.com/open-telemetry/opentelemetry-js/pull/965/commits/2ef75b5833d873f9230aaede5d3c26612ac52873